xcm/v6/transfers.json

```diff
@@ -2985,6 +3013,85 @@
               "type": "xtokens"
             }
           ]
+        },
+        {
+          "assetId": 6,
```

Comment:
Am I right that this is for Hydra? Couldn't find mention of it in the description